### ISSUE TITLE
Missing Textures for Nether broken ore/nether gravel ore

### ISSUE THREAD

**Issue by DatrixTHLK** [Jozufozu/ExNihiloOmnia#80]
What is strange is they had textures on them before I updated to forge version 2120. 1.10.2 Now I can't make Nether gravel variations of the broken nether ore. I'm also having trouble entering and exiting the nether because for some reason Nausea potion effect happens to crash minecraft after exiting or entering the nether. 

Minecraft 1.10.2
Forge: 12.18.2.2122
Exnihilo Omnia: 1.1.6

FML Log: https://gist.github.com/DatrixTHLK/2d348b9ba9d293c3504fa174978f6d3e (I will also put this FML Log up on Optifine's Issue place.

**Comment by DatrixTHLK** [comment]
[#52991](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1272953-optifine-hd-d1-fps-boost-dynamic-lights-shaders?comment=52991)

**Comment by Ravin890** [comment]
Are there even nether ores in the current version?

**Comment by Jozufozu** [comment]
Could you let me know if this happens on the [latest version](https://minecraft.curseforge.com/projects/ex-nihilo-omnia/files/2340612)?

**Comment by DatrixTHLK** [comment]
@JozuFozu Thanks for the update. I will take a look right now.

**Edit:** I will try out 1.1.6 then.

**Edit 2:** Nickel, Ardite, Lead and Cobalt Broken ore still have missing textures and aren't able to be crafted into nether gravel equivalents. Nickel, Tin, Lead and Osmium Ingots are missing their textures. Silver, Lead, Tin, Osmium and Platinum Crushed ore are missing their texture and aren't able to be crafted into sand ore equivalents. Powdered Lead and Osmium are missing their textures. I'm sure there are more missing textures than just these though. While looking into FEI's Subset menu, and looking into Ex Nihilo Omnia, these textures aren't even mentioned in the mod item list, nor are they mentioned in JEI's recipes. Which is odd because they're listed in the config file and should show up anyway.

For the most part, @Ravin890, no, Nether ore equivalents in large have no textures for them yet.

**Comment by DatrixTHLK** [comment]
I think I found out my own issue. Turns out that I had certain config settings false instead of true. I will reboot my game and see if that fixes the issue.

This is where the config was set to false instead of true: http://pastebin.com/Pq722U7d

**Edit:** As I suspected, it was user error after all. It works fine now. I will close this issue. Thanks for at least updating the  mod. I'm sorry about this.
